packages/block-library/src/navigation-link/editor.scss

```diff
@@ -80,49 +80,31 @@
 .wp-block-navigation-link__placeholder {
 	position: relative;
 
-	// Provide a little margin to show each placeholder as a separate unit.
-	margin: 2px;
+	// Appear disabled to indicate incompleteness.
+	opacity: 0.6;
```

Comment:
I'm not sure this opacity effect is going to work if we apply it to the theme's colors. In Twenty Twenty for example, it results in text that doesn't pass AA contrast: 

<img width="92" alt="Screen Shot 2021-07-07 at 12 02 13 PM" src="https://user-images.githubusercontent.com/1202812/124792885-6f56c280-df1b-11eb-9ee0-10f98ca92317.png">

I expect this would be similar in any theme that uses a text color that isn't relatively high contrast by default. Do you think this works nicely if we remove the opacity effect?

Reply:
Good thought, I removed it for now. I think it might be worth to revisit this, or a variant of this, to indicate some form of "interactivity" that feels a bit lost without it. But I'd love to do that separately.